pkg/operation/botanist/monitoring.go

```diff
@@ -84,15 +84,25 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 		b.Shoot.Components.ControlPlane.ResourceManager,
 	}
 
+	if b.Shoot.IsShootControlPlaneLoggingEnabled(b.Config) && gardenlethelper.IsValiEnabled(b.Config) {
```

Comment:
Why was `b.Shoot.IsShootControlPlaneLoggingEnabled(b.Config)` added?

Reply:
`IsShootControlPlaneLoggingEnabled ` check if logging is enabled in gardenlet config, if it is not there is no need of vali.